# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: gfx/thebes/gfxPlatformGtk.cpp
# Commit: 063622f1a9b5
# Full Hash: 063622f1a9b5bf4f4ab425fcf2ff67140d022578
# Author: Alexandre Lissy <lissyx+mozillians@lissyx.dyndns.org>
# Date: 2021-07-15 21:52:49
# Description:
#   Bug 1719668 - Do not force XFlush() on XRender r=jrmuizel,nical
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D119557
# ==============================================================================

diff -r f597999c46c0 -r 063622f1a9b5 gfx/thebes/gfxPlatformGtk.cpp
--- a/gfx/thebes/gfxPlatformGtk.cpp	Thu Jul 15 08:42:08 2021 +0000
+++ b/gfx/thebes/gfxPlatformGtk.cpp	Thu Jul 15 08:56:20 2021 +0000
@@ -258,12 +258,6 @@
   gfxVars::SetUseWebRenderCompositor(feature.IsEnabled());
 }
 
-void gfxPlatformGtk::FlushContentDrawing() {
-  if (gfxVars::UseXRender()) {
-    XFlush(DefaultXDisplay());
-  }
-}
-
 void gfxPlatformGtk::InitPlatformGPUProcessPrefs() {
 #ifdef MOZ_WAYLAND
   if (IsWaylandDisplay()) {